### DISCUSSION TITLE
Increase size of Controls_Manager::NUMBER <input> field

### DISCUSSION THREAD

**Discussion by roland-jungwirth** [elementor/elementor#18601] [Developers Community]
In my control

```
$repeater->add_control(
        'token_min', [
        'label' => __('Minimum', self::$slug),
        'type' => Controls_Manager::NUMBER,
        'default' => __('low level', self::$slug),
        'placeholder' => __('Minumim', self::$slug),
        'label_block' => TRUE,
    ]
);
```

the user can input very small numbers. The current width of the rendered input field does not suffice to show all of it, see

<img width="231" alt="elementor-input-width" src="https://user-images.githubusercontent.com/1560649/167358389-0278ac34-4b41-4a3a-99df-d08dd90c85aa.png">

As per the image, I also tried the `'label_block' => TRUE,` but that did not work. How can I increase the input field's width to take up the full width available?

**Comment by roland-jungwirth** [comment]
A possible solution would be to add some custom CSS: 

```
text-align: right; // this is for numbers
width: 100%;
```

How can I add custom CSS to my control?

**Reply by WebsiteDons** [reply]
Use the editor hook

```php
add_action( 'elementor/editor/footer', function() 
{
  echo '<style>input[type="number"] {width: 100px;}</style>';
});
```
